pkgs/applications/audio/shortwave/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, fetchFromGitLab, cargo, dbus, desktop-file-utils, gdk-pixbuf,
+  gettext, glib, gst_all_1, gtk3, libhandy, meson, ninja, openssl,
+  pkgconfig, python3, rust, rustc, rustPlatform, sqlite, wrapGAppsHook
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "shortwave";
+  version = "1.0.1";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "World";
+    repo = "Shortwave";
+    rev = version;
+    sha256 = "13lhlh75vw02vkcknl4nvy0yvpdf0qx811mmyja8bzs4rj1j9kr8";
+  };
+
+  cargoSha256 = "0aph5z54a6i5p8ga5ghhx1c9hjc8zdw5pkv9inmanca0bq3hkdlh";
+
+  nativeBuildInputs = [
+    cargo desktop-file-utils gettext glib meson ninja pkgconfig
```

Comment:
Also `pkg-config` should be used in new expressions.

Reply:
It would be also nicer to format the dependencies one per line to make the future diffs cleaner.